.github/workflows/allennlp.yml

```diff
@@ -0,0 +1,39 @@
+name: allennlp
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+    paths:
+      - 'allennlp/**'
+      - '.github/workflows/allennlp.yml'
+
+jobs:
+  examples:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    if: github.repository == 'optuna/optuna-examples'
```

Comment:
In the future, it might make sense to remove this check given https://github.com/optuna/optuna/pull/2656. I also noticed that PyTorch has a disclaimer at the top of their README to disable daily when forking at the user side.

```
WARNING: if you fork this repo, github actions will run daily on it. To disable this, go to /examples/settings/actions and Disable Actions for this repository
```

https://github.com/pytorch/examples